src/BIMDataComponents/BIMDataMenu/BIMDataMenu.vue

```diff
@@ -137,71 +114,39 @@ export default {
       type: String,
       default: "200px",
     },
-    isClickAway: {
-      type: Boolean,
-      default: true,
-    },
     childrenLeft: {
       type: Boolean,
       default: false,
     },
   },
-  emits: ["hover"],
```

Comment:
It looks like you're still emitting an  `"hover"` event in the `onMouseOver` method below.
You should keep the `emits` options then.